src/main/java/ports/usecases/account/removeAchiveCourse/RemoveArchiveCourseInputBoundary.java

```diff
@@ -0,0 +1,15 @@
+package ports.usecases.account.removeAchiveCourse;
+
+import ports.usecases.PathNotFoundError;
+import ports.usecases.account.archiveCourse.ArchiveCourseInputBoundary;
+
+public interface RemoveArchiveCourseInputBoundary {
+    /**
+     * Send remove a course from this account's archive.
+     * @param username the username of the account
+     * @param courseCode the name of the course that the user wants to remove
+     * @throws PathNotFoundError if the username or courseCode do not exist
+     */
+
+    void execute(String username, String courseCode) throws PathNotFoundError, ArchiveCourseInputBoundary.CourseNotCompletedError;
```

Comment:
There's no reason for this method to throw a `CourseNotCompletedError`, since if a course is already in your archive it should be completed.

Reply:
I checked over the implementation, and it doesn't throw this error. So it's safe to remove it from the declaration here.